sdk/services/exchange.ts

```diff
@@ -663,12 +663,17 @@ export default class ExchangeService {
 		]);
 
 		if (!!quoteCurrencyContract) {
-			const allowance = (await quoteCurrencyContract.allowance(
-				this.sdk.context.walletAddress,
-				approveAddress
-			)) as ethers.BigNumber;
-
-			return wei(ethers.utils.formatEther(allowance));
+			try {
+				const allowance = (await quoteCurrencyContract.allowance(
+					this.sdk.context.walletAddress,
+					approveAddress
+				)) as ethers.BigNumber;
+
+				return wei(ethers.utils.formatEther(allowance));
+			} catch (e) {
+				// eslint-disable-next-line no-console
+				console.error('checkAllowance: ', e);
```

Comment:
Could you please use our own `logError` function here? It's easy to manage these error messages together in the future. https://github.com/Kwenta/kwenta/blob/main/utils/logError.ts#L1